dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -507,6 +507,11 @@ public Optional<Rule> getRule(final Experiment experiment)
         return UtilMethods.isSet(rules) ? Optional.of(rules.get(0)) : Optional.empty();
     }
 
+    @Override
+    public boolean isAnyExperimentRunning() throws DotDataException {
+        return !APILocator.getExperimentsAPI().getRunningExperiment().isEmpty();
```

Comment:
I think we need to use cache here, we should create another issue to add the cache, thoughts?